build/bin/check-pci-get-cells

```diff
@@ -25,7 +25,7 @@ import check_cli
 expected_headers = ['ID', 'Node', 'ID', 'Dlearfcn', 'Cell', 'Type', 'PCI', 'PCI', 'Pool']
 
 ID_REGEX = r'[0-9a-f]{15}'
-NODE_ID_REGEX = r'[0-9a-f]+'
+NODE_ID_REGEX = r'e2:[0-9a-f/-]+'
```

Comment:
Should (this)[https://github.com/onosproject/onos-test/blob/master/build/bin/check-topo-entities#L25] be changed to match?

Reply:
Fixed this; not the other. Good catch.